### ISSUE TITLE
Mini-Map

### ISSUE THREAD

**Issue by BUNN1E5** [AztecGameLab/LongTermGame5#27]
Lets try with simple looking map

Maybe something like this?
~~https://raw.githubusercontent.com/alejandro61299/Minimaps_Personal_Research/master/docs/web_images/lol.png~~

**Comment by hlu5118** [comment]
I was thinking a minimap like in terraria

**Comment by BUNN1E5** [comment]
So I was looking at those minimaps, I think maybe a metroid style minimap might be better cause it doesn't give to much extra information

![image](https://user-images.githubusercontent.com/6379273/108577010-6949a780-72d4-11eb-832f-373d24e666c6.png)
![image](https://user-images.githubusercontent.com/6379273/108577033-7797c380-72d4-11eb-8ee7-614cec5d5f1c.png)
![image](https://user-images.githubusercontent.com/6379273/108577044-82eaef00-72d4-11eb-8256-c16ce11427db.png)

**Comment by hlu5118** [comment]
good idea, i think a minimap should be simple and easy for the player to understand just by looking at it

**Comment by chmilk1** [comment]
I am taking the level loading system's bounding boxes to procedurally generate the minimap so art doesn't have to make a map
